src/libCom/yajl/yajl_parser.c

```diff
@@ -45,6 +45,8 @@
 #include "yajl_encode.h"
 #include "yajl_bytestack.h"
 
+#include <osdStrtod.h>
+
```

Comment:
I'd prefer that you replace the `#include <stdlib.h>` with `#include <epicsStdlib.h>` than pull in this header, which was really meant to be an implementation detail.

Reply:
Thanks - sorry, I should have looked more closely how it was used elsewhere in the code